From 3ac051c4549575634cecc706175b019f4ed4c3bf Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Sat, 22 Jul 2017 22:47:05 +0200
Subject: Fix plugins build dependency

Do not build the plugins before the dependency on module positioning for
the plugin subdir position is available.

Task-number: QTBUG-62098
Change-Id: Ic7473db937359f3dad47aef0ffa3671be8cbd349
Reviewed-by: Alex Blasche <alexander.blasche@qt.io>

Upstream: https://code.qt.io/cgit/qt/qtlocation.git/patch/?id=3ac051c4549575634cecc706175b019f4ed4c3bf
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 src/src.pro | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/src.pro b/src/src.pro
index d0a1ee4e7..0c0bf50b1 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -7,8 +7,6 @@ SUBDIRS += positioning
 positioning.depends = 3rdparty/clip2tri
 
 qtHaveModule(quick) {
-    plugins.depends += positioning
-
     SUBDIRS += location
     location.depends += positioning 3rdparty/clip2tri
 
@@ -18,6 +16,7 @@ qtHaveModule(quick) {
     imports.depends += positioning location
 }
 
+plugins.depends += positioning
 SUBDIRS += plugins
 
 !android:contains(QT_CONFIG, private_tests) {
-- 
cgit v1.1-6-g87c4

